htdp-lib/htdp/bsl/reader.rkt

```diff
@@ -26,15 +26,18 @@
 (define ((make-info options) key default use-default)
   (case key
     [(drscheme:toolbar-buttons)
+     ;; If you want to add more buttons, don't do it this way:
```

Comment:
Maybe we should write something like:

```
;; if you want to add more buttons, do it this way only 
;; if they aren't already built and can just be enabled.
```

?

Reply:
I had trouble parsing this comment; I think you mean 
"aren't already (built and can just be enabled)", but I initially read it as "(aren't already built) and (can just be enabled)"... which doesn't really make sense, but maybe something like "aren't already built and enable-able" would be less misunderstandable. Oog.

Reply:
How's this:

```
;; if you want to add more buttons, the strategy used here might not
;; be the right one to use. If you can simply enable existing buttons,
;; do that instead.
```

Reply:
@rfindler Done.